### ISSUE TITLE
Create account feature

### ISSUE THREAD

**Issue by GWill44** [GWill44/login-page#2]
Given I am a user, when I land on the login page, then the option to login or create an account is displayed.
When I click on create account, then a account creation form is displayed.
When the form is completed and the create button is clicked, then I am returned to the login page.

Acceptance criteria:
- [x] Display a sign up button next to the login button.
- [x] Display an account creation form that opens on 'sign up' button click.
- [x] Check that password and password confirmation match.
- [x] Return user to the login page once account creation is complete.

**Comment by GWill44** [comment]
@coeymusa There is a bug in the field input verification. If a field is empty its respective prompt is displayed at the bottom right of the registration modal. This works for all fields besides the date of birth. Could you take a look at this please?
[](https://github.com/GWill44/login-page/tree/registration-feature)

**Comment by coeymusa** [comment]
Created new issue: https://github.com/GWill44/login-page/issues/4
